geopandas/geoseries.py

```diff
@@ -152,8 +152,6 @@ class GeoSeries(GeoPandasBase, Series):
 
     """
 
-    _metadata = ["name"]
```

Comment:
I think this is a left-over from the time that "crs" was part of this list, so we needed to override this. But now we don't have custom attributes anymore for GeoSeries, so we don't need to override this (it's actually also wrong, because pandas nowadays has `_name` instead of `name` as metadata item)